components/dashboard/src/admin/WorkspaceDetail.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+import { User, WorkspaceAndInstance } from "@gitpod/gitpod-protocol";
+import moment from "moment";
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { getGitpodService } from "../service/service";
+import { getProject, WorkspaceStatusIndicator } from "../workspaces/WorkspaceEntry";
+import { getAdminLinks } from "./gcp-info";
+import { Property } from "./UserDetail";
+
+export default function WorkspaceDetail(props: { workspace: WorkspaceAndInstance }) {
+    const [workspace, setWorkspace] = useState(props.workspace);
+    const [activity, setActivity] = useState(false);
+    const [user, setUser] = useState<User>();
+    useEffect(() => {
+        getGitpodService().server.adminGetUser(props.workspace.ownerId).then(setUser);
+    }, [props.workspace]);
+
+    const stopWorkspace = async () => {
+        try {
+            setActivity(true);
+            await getGitpodService().server.adminForceStopWorkspace(workspace.workspaceId);
+            // let's reload in a sec
+            setTimeout(reload, 2000);
+        } finally {
+            setActivity(false);
+        }
+    }
+
+    const reload = async () => {
+        try {
+            setActivity(true);
+            const ws = await getGitpodService().server.adminGetWorkspace(workspace.workspaceId);
+            setWorkspace(ws);
+        } finally {
+            setActivity(false);
+        }
+    }
+    const adminLinks = getAdminLinks(workspace);
+    const adminLink = (i: number) => <Property key={'admin-'+i} name={adminLinks[i]?.name || ''}><a className="text-blue-400 hover:text-blue-600" href={adminLinks[i]?.url}>{adminLinks[i]?.title || ''}</a></Property>;
+    return <>
+        <div className="flex">
+            <div className="flex-1">
+                <div className="flex"><h3>{workspace.workspaceId}</h3><span className="my-auto ml-3"><WorkspaceStatusIndicator instance={WorkspaceAndInstance.toInstance(workspace)} /></span></div>
+                <p>{getProject(WorkspaceAndInstance.toWorkspace(workspace))}</p>
+            </div>
+            <button className="secondary ml-3" disabled={activity} onClick={reload}>Reload Data</button>
```

Comment:
**question**: No more _Download Workspace_ button?

Reply:
It doesn't work AFAIK